docker-compose.yml

```diff
@@ -15,10 +15,11 @@ services:
       context: ./backend
       dockerfile: Dockerfile
     environment:
-      - PRIVATE_KEY=0
-      - STARKNET_ACCOUNT_ADDRESS=0
-      - TOKEN_ADDRESS=0
-      - RPC_URL=0
+      - PRIVATE_KEY=0x0300001800000000300000180000000000030000000000003006001800006600
+      - STARKNET_ACCOUNT_ADDRESS=0x03ee9e18edc71a6df30ac3aca2e0b02a198fbce19b7480a63a0d71cbd76652e0
+      - TOKEN_ADDRESS=0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7
+      - KAKAROT_ADDRESS=0x66a5454898f533792a1b3563d808c680a1cf6dd485797fc7044d71d88c49205  
+      - RPC_URL=http://host.docker.internal:5050
```

Comment:
this assumes a dev setup, need to be mindful of how its set on deploy

Reply:
can these values be overridden a launch by the docker compose?
for instance if kakarot_address changes, or even if rpc is run on madara for instance

Reply:
we may want to use env variable here instead, and https://docs.docker.com/compose/environment-variables/set-environment-variables/, and create env file for katana, madara, other networks with more private info, etc.